src/assets/html/index.html

```diff
@@ -8,7 +8,37 @@
   </head>
   <body>
     <h1>プチ成果物集</h1>
-    <h2>ToDoリスト</h2>
+    <section>
+      <h2>ToDoリスト</h2>
+      <section>
+        <h3>新規タスクの追加</h3>
+        <label>タスク名</label>
+        <input type="text" data-input="taskname" />
+        <label>期限</label>
+        <input type="date" data-input="deadline" />
```

Comment:
JSではないですがa11y観点で`label`要素は下記のようにすることで`input`属性との紐づけが行われます。

```suggestion
        <label>
          期限
          <input type="date" data-input="deadline" />
        </label>
```

Reply:
ありがとうございます〜！
せっかくなのでlabelの中にinputを内包する形に変更しました！
ea6249079b7b8f94defba70b89ed349144162394

Reply:
👍 いただいているのでresolveします！